### PR TITLE
Add Picket Auth Integration Guide

### PR DESCRIPTION
**Pull request by devstein**
Hi 👋 

## What kind of change does this PR introduce?

Documentation on how to integrate [Picket](https://picketapi.com/) with Supabase to enable wallet login for users that want to build hybrid web 2/3 apps. The new docs are an adaptation of the existing [Picket + Supabase docs](https://docs.picketapi.com/picket-docs/reference/integrations/supabase) in the [Picket documentation](https://docs.picketapi.com/).

#### Common Use-Case for Picket + Supabase
-   Account linking. Allow users to associate their wallet address(es) with their existing web2 account in your app
-   Leverage Supabase's awesome libraries and ecosystem while still enabling wallet login
-   Store app-specific data, like user preferences, about your user's wallet adress off-chain
-   Cache on-chain data to improve your DApp's performance

## What is the current behavior?

N/A

## What is the new behavior?

N/A

## Additional context

I've been emailing @thorwebdev

### CODE REVIEW

**Comment by thorwebdev** [comment]
@devstein thanks so much for this. Quick heads up that we're sprinting towards [Launch Week](https://supabase.com/launch-week) right now and might be a bit delayed in reviewing this, sorry!

**Review by thorwebdev — CHANGES_REQUESTED** [review]
Some changes needed to apply the page styling correctly.

File: apps/docs/components/Navigation/Navigation.constants.ts [hunk 1]
```diff
@@ -296,6 +296,7 @@ export const menuItems: NavMenu = {
             { name: 'keyri', url: '/guides/integrations/keyri', items: [] },
             { name: 'Stytch', url: '/guides/integrations/stytch', items: [] },
             { name: 'SuperTokens', url: '/guides/integrations/supertokens', items: [] },
+            { name: 'Picket', url: '/guides/integrations/picket', items: [] },
```
[the remark this answers is not in the document]
Comment (devstein) [reply apps/docs/components/Navigation/Navigation.constants.ts]: done

**Review by thorwebdev — APPROVED** [review]
LGTM, thanks.

**Comment by thorwebdev** [comment]
@devstein sorry, the prettier check is failing. Can you either run prettier or allow me to push to your repo?

**Comment by devstein** [comment]
Hi @thorwebdev I ran prettier on `picket.mdx`, but if I run the repo-level `npm run format` the diff is quite large. Regardless, gave you write access to the repo. 

I also noticed that the `package.json` command doesn't include `.mdx` files but the CI check appears to

```json
    "format": "prettier --write \"apps/**/*.{js,jsx,ts,tsx,css,md,json}\"",
```

Thanks!

**Pull request merged**